README.md

```diff
@@ -35,6 +35,7 @@ Each cafe will be handpicked and “features” ranked like wifi signal, how gre
 - [Nori] (https://github.com/denesnori) - Back End, Front End
 - [Raquel](https://github.com/raquel-ucl) - Back End, Front End
 - [Nuria](https://github.com/nuria-gs) - Learning
+- [Emiliana](https://github.com/emilianas) - Learning
```

Comment:
Could you please add in Backend or Frontend instead of learning so we know what side of the project you would like to help out on? If can be both :)